crates/libcontainer/src/rootfs/mount.rs

```diff
@@ -171,10 +171,26 @@ impl Mount {
         tracing::debug!("cgroup mounts: {:?}", host_mounts);
 
         // get process cgroups
+        let pid1_cgroups = Process::new(1)?.cgroups()?.0;
```

Comment:
If users don't separate the PID namespace, what happens?

Reply:
Good point!
It fails in the same way as without the fix :-(

Reply:
When not unsharing PID namespace, using the parent PID does the trick :-)
I just pushed a fix